sfge/src/test/java/com/salesforce/graph/symbols/StaticCodeBlockInvocationTest.java

```diff
@@ -0,0 +1,300 @@
+package com.salesforce.graph.symbols;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.salesforce.TestRunner;
+import com.salesforce.TestUtil;
+import com.salesforce.graph.symbols.apex.ApexStringValue;
+import com.salesforce.graph.symbols.apex.ApexValue;
+import com.salesforce.graph.visitor.SystemDebugAccumulator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class StaticCodeBlockInvocationTest {
```

Comment:
Tests to verify static block invocation while setting static scope of class. I've added and disabled test cases that need future work.